editor.planx.uk/src/@planx/components/PropertyInformation/PropertyInformation.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import { MockedProvider } from "@apollo/client/testing";
+import Wrapper from "@planx/components/fixtures/Wrapper";
+import { Meta, StoryObj } from "@storybook/react";
+import React from "react";
+
+import Editor from "./Editor";
+import { Presentational, PresentationalProps } from "./Public";
+
+export default {
+  title: "PlanX Components/PropertyInformation",
+  component: Presentational,
+} as Meta;
+
+// Would be nice to import/share these default props between tests & stories, but Storybook will throw error "jest is not defined" on overrideAnswer
```

Comment:
It's not perfectly neat but could we do something like this, which may express and expose the issue we're facing here.

```ts
const defaultPresentationalProps: PresentationalProps = merge(
   objectUsedByJestTests,
   { overrideAnswer: () => console.log("test") }
   // or make an object if there are multiple overrides maybe?
   // objectWithStorybookOverrides,
 )
```

Reply:
Addressed now, both tests & stories pulling from same mock file and each setting appropriate `overrideAnswer` :+1: